tests/e2e-disabled/restart-node-multi-sc/setup-vdb/base/setup-vdb.yaml

```diff
@@ -17,6 +17,9 @@ metadata:
   name: v-mc-restart
 spec:
   image: replace-with-kustomize
+  sidecars:
+    - name: vlogger
+      image: kustomize-vlogger-image
   communal:
```

Comment:
All these repeated modifications almost makes one think there should be a yaml template somehow used to generate all the yaml file boilerplate.